db/migrate/20200923192959_create_cavc_remand.rb

```diff
@@ -0,0 +1,22 @@
+class CreateCavcRemand < Caseflow::Migration
+  def change
+    create_table :cavc_remands do |t|
+      t.bigint "appeal_id", null: false, comment: "Appeal the court is remanding"
+      t.bigint "created_by_id", null: false, comment: "User that created this record"
+      t.bigint "updated_by_id", null: false, comment: "User that updated this record. For MDR remands, judgement and mandate dates will be added after the record is first created."
+
+      t.string "cavc_docket_number", null: false, comment: "Docket number of the CAVC judgement"
+      t.boolean "represented_by_attorney", null: false, comment: "Whether or not the appellant was represented by an attorney"
+      t.string "cavc_judge_full_name", null: false, comment: "CAVC judge that passed the judgement on the remand"
+      t.string "cavc_decision_type", null: false, comment: "CAVC Decision type. Currently one of 'Remand', 'Straight Reversal', and 'Death Dismissal'"
```

Comment:
```suggestion
      t.string "cavc_decision_type", null: false, comment: "CAVC decision type. Expecting 'remand', 'straight_reversal', or 'death_dismissal'"
```
You could also reference the `CAVC_DECISION_TYPES.json` file instead of listing the possible values.